iphone/Classes/TiUIAlertDialogProxy.m

```diff
@@ -35,7 +35,9 @@ -(NSMutableDictionary*)langConversionTable
 			@"title",@"titleid",
 			@"ok",@"okid",
 			@"message",@"messageid",
-			nil];
+            @"loginPlaceholder", @"passwordPlaceholder",
```

Comment:
What is the purpose of adding these 3 new objects here? This is a dictionary with Objects and Keys, there's no corresponding keys for these objects.